app/services/feature_toggle.rb

```diff
@@ -1,7 +1,11 @@
 class FeatureToggle
   # Disable check on preview apps to allow for testing of future forms
   def self.saledate_collection_window_validation_enabled?
-    true
+    Rails.env.production? || Rails.env.test? || Rails.env.staging?
```

Comment:
have you checked this won't have any unexpected changes on local envs?

Reply:
Good point! It deffo removes the validation locally as well. It doesn't matter too much cause we can always toggle things on/off locally ourselves anyway, so any preference on this? I think it aligns with some other toggles we have atm

Reply:
yeah fine as is by me :D